### PR TITLE
Single cimetrics

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

**Comment by cimetrics** [comment]
single_cimetrics@[3505 aka 20191217.7](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3505&view=results) vs master ewma over 20 builds from [3303](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3303&view=results) to [3493](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3493&view=results)
images

**Comment by cimetrics** [comment]
single_cimetrics@[3509 aka 20191217.9](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3509&view=results) vs master ewma over 20 builds from [3303](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3303&view=results) to [3493](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3493&view=results)
images

File: .azure-pipelines-templates/cimetrics.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+jobs:
+- job: CIMetrics
+  displayName: 'CI Metrics'
+  
+  ${{ insert }}: ${{ parameters.env }}
+
+  steps:
+    - checkout: self
```
Comment (achamayou) [inline .azure-pipelines-templates/cimetrics.yml]: It's mildly annoying to do this just to get metrics.yml, but I don't have a better idea.

**Comment by cimetrics** [comment]
single_cimetrics@[3511 aka 20191217.10](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3511&view=results) vs master ewma over 20 builds from [3303](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3303&view=results) to [3493](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3493&view=results)
images

**Pull request merged**